### PR TITLE
add filter cases

### PR DESCRIPTION
**Pull request by wpride**

### CODE REVIEW

**Comment by wpride** [comment]
@benrudolph let's fire this fucker up!

**Comment by wpride** [comment]
:fire:

File: corehq/apps/cloudcare/touchforms_api.py [hunk 1]
```diff
@@ -74,9 +74,14 @@ def filter_cases(self, xpath, additional_filters=None, auth=None, extra_instance
             "session_data": session_data,
         }
 
+        if use_formplayer:
+            url = settings.FORMPLAYER_URL + "/filter_cases"
+        else:
+            url = settings.XFORMS_PLAYER_URL
```
Comment (wpride) [inline corehq/apps/cloudcare/touchforms_api.py]: @benrudolph curious if you think we should continue having the action contained in the content body or start routing the different requests to different URLs

Comment (benrudolph) [inline corehq/apps/cloudcare/touchforms_api.py]: i think we can drop the action. also this url thing is going to get a bit out of hand with all these hardcoding of strings and we can't use Django's reverse method. might be worth making a utility function that takes an action and returns a url (maybe located in `corehq/apps/cloudcare/util.py`). then we can call `formplayer_url(action)` and it'll neatly return the proper URL

**Comment by benrudolph** [comment]
:smoking: hot. watch :earth_americas:

**Comment by czue** [comment]
:snowflake: :rugby_football: s

**Pull request merged**